### PR TITLE
Create script for cherry picking to release branch

### PR DESCRIPTION
**Pull request by brettchabot**

### CODE REVIEW

**Review by AkshDesai04 — APPROVED** [review]
Can't we already fire a git status instead of telling the user to do so?

**Review by AkshDesai04 — CHANGES_REQUESTED** [review]
Adding a call for git status ourselves would make it more functional.

`if [[ -n $(git status -s) ]]; then`
`    echo "Error: git directory is not clean; Running 'git status'"`
`    git status`
`    exit 1`
`fi`

File: tools/release/cherrypick_to_release_branch.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    exit 1
+fi
+
+if [ "$#" -ne 1 ]; then
+    echo "Error: Unexpected number of parameters: Usage: cherrypick_to_release_branch.sh commit_hash"
+    exit 1
+fi
+
+if [[ ! "$RELEASE_BRANCH" =~ "release_branch" ]]; then
+    echo "Error: RELEASE_BRANCH env var is undefined or does not contain 'release_branch'"
```
Comment (TWiStErRob) [inline tools/release/cherrypick_to_release_branch.sh:23]: output the value to show which one?
```suggestion
    echo "Error: RELEASE_BRANCH env var (${RELEASE_BRANCH}) is undefined or does not contain 'release_branch'"
```

File: tools/release/cherrypick_to_release_branch.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#
+#
+# Usage:
+# export RELEASE_BRANCH=axt_XXX_release_branch # if not already defined
+# bash tools/release/cherrypick_to_release_branch.sh commit_hash
+
+set -e
+
+if [[ -n $(git status -s) ]]; then
+    echo "Error: git directory is not clean; check 'git status'"
```
Comment (AkshDesai04) [inline tools/release/cherrypick_to_release_branch.sh:13]:
```suggestion
    echo "Error: git directory is not clean; running 'git status'"
    git status
```

**Pull request merged**